File: pset6/mario.py
height = 0
while height < 1 or height > 8:
    height = input("Height: ")  # ask height and check value
    try:
        height = int(height)
    except:
        height = 0

for i in range(1, height + 1):
    print(" " * (height - i), "#" * i, "  ", "#" * i, sep="")  # print pyramid

File: pset1/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{   
    int height;
    do
    {
        height = get_int("Height: ");
    }
    while (height < 1 || height > 8); //Check input number gor height
    for (int strings = 0; strings < height; strings++) //Draw strings
    {
        for (int col = 0; col < height - strings - 1; col++) //Draw space before
        {
            printf(" ");
        }
        for (int col = 0; col < strings + 1; col++) //Draf first line of blocks
        {
            printf("#");
        }
        printf("  "); //Draw space between lines of blocks
        for (int col = 0; col < strings + 1; col++) //Draw second line of blocks
        {
            printf("#");
        }
        printf("\n");
    }
}
